request_public_data.py

```diff
@@ -23,16 +25,19 @@
 if args.verbose:
 	logging.basicConfig(level=logging.DEBUG)
 
-r = requests.get("http://" + server_url + "/getData?from_date=" + from_date + "&to_date=" + to_date)
+from_date_UTC = arrow.get(from_date).replace(tzinfo='local').to('utc').format('YYYY-MM-DD HH:mm:ss')
```

Comment:
I am not sure that this actually parses the data correctly. IIRC, you need to do

`arrow.get(year=xx, month=xx, day=xx, tzinfo='local')`
then you can just do `.timestamp` on that.

I'd suggest some poking around with arrow in an ipython notebook